### DISCUSSION TITLE
ioredis

### DISCUSSION THREAD

**Discussion by MR4online** [DefinitelyTyped/DefinitelyTyped#53736] [Issues with a @types package]
**incomple @types/ioredis**

this does work in JS:

```
ioRedis.multi()
    .send_command('JSON.SET', [key, ".id", "45"])
    .exec((err, results) => {});
```

but **send_command** is not recognized on pipelines in TS, so I have to `@ts-ignore` it to get it working.

----
May be you can update this?
@plantain-00 @chrisleck @aliarham11
Thanks!

**Comment by plantain-00** [comment]
You can update it and create a PR
